conf/defaults.ini

```diff
@@ -126,7 +126,8 @@ connstr =
 # This enables data proxy logging, default is false
 logging = false
 
-# How long the data proxy should wait before timing out default is 30 (seconds)
+# How long the data proxy should wait before timing out default is 30 (seconds).
+# Note: This setting also applies to core backend HTTP data sources where queries are requested using an HTTP client with timeout set.
```

Comment:
```suggestion
# This setting also applies to core backend HTTP data sources where queries requests use an HTTP client with timeout set.
```

Reply:
If you accept this suggestion, please apply everywhere it is used.

Reply:
```suggestion
# This setting also applies to core backend HTTP data sources where query requests use an HTTP client with timeout set.
```